src/mixins/filing-template-mixin.ts

```diff
@@ -986,12 +922,101 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
   // Local helper methods
   //
 
+  /** The Contact Point object. */
+  private get getContactPoint (): ContactPointIF {
+    return {
+      email: this.getBusinessContact.email,
+      phone: this.getBusinessContact.phone || '',
+      extension: +this.getBusinessContact.extension || undefined // don't include if empty
+    }
+  }
+
+  /**
+   * Adds the completing party into the specified parties array.
+   * Assumes the completing party doesn't already exist in the array.
+   * @param parties the parties array
+   * @returns the updated parties array
+   */
+  private addCompletingParty (parties: OrgPersonIF[]): OrgPersonIF[] {
+    parties.push({
+      officer: {
+        partyType: PartyTypes.PERSON,
+        firstName: this.getCompletingParty.firstName,
+        middleName: this.getCompletingParty.middleName,
+        lastName: this.getCompletingParty.lastName
+      },
+      roles: [{
+        roleType: RoleTypes.COMPLETING_PARTY,
+        appointmentDate: this.getCurrentDate
+      }],
+      mailingAddress: this.getCompletingParty.mailingAddress
+    } as OrgPersonIF)
+
+    return parties
```

Comment:
Since objects are pass-by-reference, this method acts on the original object.

But it's nice to return the updated object just in case, or for consistent code. (It's up to the caller whether to look at the return value.)